python/ccxt/base/exchange.py

```diff
@@ -6765,3 +6769,65 @@ def clean_cache(self, subscription: dict):
                     tickerSymbol = tickerSymbols[i]
                     if tickerSymbol in self.tickers:
                         del self.tickers[tickerSymbol]
+
+    def get_zk_contract_signature_obj(self, seeds: str, params={}):
```

Comment:
these methods should be added before these lines to avoid being transpiled 
![image](https://github.com/user-attachments/assets/e97965ab-5d73-4925-a4e3-2fd343b0e059)